<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QSplitter" name="splitter">
                        <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="handleWidth">
                            <number>2</number>
                        </property>
                        <widget class="QTabWidget" name="navigationTabWidget">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="minimumSize">
                                <size>
                                    <width>150</width>
                                    <height>0</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>300</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="baseSize">
                                <size>
                                    <width>200</width>
                                    <height>0</height>
                                </size>
                            </property>
                            <property name="currentIndex">
                                <number>0</number>
                            </property>
                            <widget class="QWidget" name="mainTab">
                                <attribute name="title">
                                    <string>Main</string>
                                </attribute>
                            </widget>
                            <widget class="QWidget" name="filteredTab">
                                <attribute name="title">
                                    <string>Filtered</string>
                                </attribute>
                            </widget>
                        </widget>
                        <widget class="QTabWidget" name="notesTabWidget">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                        </widget>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
